brayns/common/renderer/Renderer.h

```diff
@@ -31,14 +31,12 @@ class Renderer
 public:
     BRAYNS_API Renderer(ParametersManager& parametersManager);
     virtual ~Renderer() {}
-    virtual void render(FrameBufferPtr frameBuffer) = 0;
+    virtual float render(FrameBufferPtr frameBuffer) = 0;
```

Comment:
I don't think render() should return a float. It's not obvious what it means, and it's not even documented. I would suggest having a separate getter for that float value.